lms/static/scripts/frontend_apps/utils/test/vitalsource-test.js

```diff
@@ -0,0 +1,41 @@
+import { bookIDFromURL } from '../vitalsource';
+
+describe('#bookIDFromURL', () => {
```

Comment:
```suggestion
describe('bookIDFromURL', () => {
```

This is just a free function rather than a method.